### DISCUSSION TITLE
Place for unit test results to be archived

### DISCUSSION THREAD

**Discussion by stefanhaerter** [RotherOSS/otobo#2535] [Ideas]
Our unit test command, `bin/otobo.Console.pl Dev::UnitTest::Run`, produces detailed output which can be made persisting easily by piping it to a file. @bschmalhofer has already tracked remaining unit test failures within various issues.

I thought about some kind of archive to store unit test result files, at best together with used parameters, git branch and commit data etc. - the purpose(s) would be e.g. to easily view when the last time was that someone ran the test suite with a certain configuration or to track down when a certain error occurred first.

My first idea was a dedicated issue here on GitHub, but this would become bloated and confusing within nearly no time, I suppose. But as I consider this to be a potentially promising idea, I wanted to leave this here as a discussion.

**Comment by bschmalhofer** [comment]
Of course this test run aggregation could be combined with continous integration and test run aggregation like in http://matrix.cpantesters.org/?dist=DBI .

**Comment by stefanhaerter** [comment]
This looks like a great tool, cool idea!
